.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
   - "4.2"
-  - "5"
+  - "5.5"
```

Comment:
By being more specific, we will be locked to that minor version, and we won't be testing in > `5.5`, once those versions become available on the servers. Was that your intention?